id-reservations/item-ids.json5

```diff
@@ -14,5 +14,6 @@
   [ -5000,  -5999, "Captain_Wonders"],
   [ -6000,  -6999, "AnnoyingPerson"],
   [ -7000,  -7999, "Cyph"],
+  [ -8000,  -8999, "Old-FW"],
   [-69000, -69999, "Penkly"],
-]
+  ]
```

Comment:
Thanks for the change. However, also this brace moved, which was not supposed to. The actual reservation line is perfect!

Reply:
When I go to edit the file it looks good.

[
  // Legacy reservations, deprecated
  [8890100, 8890150, "Sinai"],
  [5850750, 5850750, "Sinai"],
  [2800900, 2800902, "Sinai"],
  [2501000, 2502999, "ehaugw"],

  // active reservations"
  [     0,   -999, "ehaugw"],
  [ -1000,  -1999, "IggyTheMad"],
  [ -2000,  -2999, "Saint"],
  [ -3000,  -3999, "raphendyr"],
  [ -4000,  -4999, "Kek"],
  [ -5000,  -5999, "Captain_Wonders"],
  [ -6000,  -6999, "AnnoyingPerson"],
  [ -7000,  -7999, "Cyph"],
  [ -8000,  -8999, "Old-FW"],
  [-69000, -69999, "Penkly"],
  ]

I don't know what to change here.

Reply:
No problem. I merged this manually and fixed that :)

Reply:
Thanks a lot.